atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -68,6 +68,8 @@ class KojiUploadPlugin(PostBuildPlugin):
     key = PLUGIN_KOJI_UPLOAD_PLUGIN_KEY
     is_allowed_to_fail = False
 
+    args_from_user_params = map_to_user_params("koji_upload_dir", "platform")
+
     def __init__(self, tasker, workflow, koji_upload_dir, blocksize=None,
                  platform='x86_64', report_multiple_digests=False):
```

Comment:
you can also remove report_multiple_digests arg since that isn't used anymore

Reply:
Good catch, deleted